Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -350,6 +351,9 @@ def gsvd(M, Wu, Wv):
     F = numpy.c_[D_r] * U * D
     G = numpy.c_[D_c] * V.T * D
 
+    if F.shape == (1, 1) and F == numpy.array([[0]]):
```

Comment:
I had to read this twice... Why not `if F.shape == (1, 1) and F[0, 0] == 0`? Simpler and less work for all. Also, F[0, 0] = 1 below. Sorry for overlooking this before.